src/menu.rb

```diff
@@ -0,0 +1,175 @@
+require './book'
+require './rental'
+require './student'
+require './teacher'
+require './persons'
+
+class App
+  def initialize
+    @book_list = []
+    @people_list = []
+    @rental_list = []
+    @user_menu = "
+			Please choose an option by entering a number:
+			1 - List all books
+			2 - List all people
+			3 - Create a person
+			4 - Create a book
+			5 - Create a rental
+			6 - List all rentals for a given person id
+			7 - Exit
+		"
+  end
+
+  def menu
+    loop do
+      puts @user_menu
+      @input = gets.chomp.to_i
+      users_selection(@input)
+      break if @input == 7
+    end
+  end
+
+  def users_selection(users_input)
+    case users_input
+    when 1
+      list_books
+    when 2
+      all_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_all_rentals_by_id
+    else
+      puts 'Thank you for using this app'
+    end
+  end
+
+  def list_books
+    if @book_list.empty?
+      puts 'There are no book in the library yet!!'
+    else
+      @book_list.each do |book|
+        puts "Title: #{book.title}\n Author: #{book.author}\n"
+      end
+    end
+  end
+
+  def all_people
+    if @people_list.empty?
+      puts 'There are not people yet'
+    else
+      @people_list.each do |person|
+        puts "Name: #{person.name}\n Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_student
```

Comment:
- [ ] I tried creating a student and a teacher and I am getting an error, kindly fix that please.
![image](https://user-images.githubusercontent.com/94551484/216785894-4635bdd7-7734-4a6d-9b82-2c4ecf569c85.png)